pkg/networkutils/network.go

```diff
@@ -571,15 +573,15 @@ func (n *linuxNetwork) buildIptablesConnmarkRules(vpcCIDRs []string, ipt iptable
 	}
 
 	var iptableRules []iptablesRule
-	log.Debugf("Setup Host Network: iptables -t nat -A PREROUTING -i %s+ -m comment --comment \"AWS, outbound connections\" -m state --state NEW -j AWS-CONNMARK-CHAIN-0", n.vethPrefix)
+	log.Debugf("Setup Host Network: iptables -t nat -A PREROUTING -i %s+ -m comment --comment \"AWS, outbound connections\" -j AWS-CONNMARK-CHAIN-0", n.vethPrefix)
```

Comment:
For the old rule with redundant state NEW, we can force delete. Append the old rule with shouldExist field set to false.

Tests:
- Add a new UT with old rule, verify the old one gets deleted
- Update the remaining UTs to refer to the new rule
- Upgrading from old ds to new, old rule should not be there
- once the new rules get added, restarting the DS should be fine - no errors in the logs